ckanext/dgu/theme/templates/page.html

```diff
@@ -15,18 +15,16 @@
 #}
 {%- block content %}
     {% block maintag %}<div role="main">{% endblock %}
-      <div id="content">
-        {% block main_content %}
```

Comment:
Ping @pduchesne - Sorry, maybe you didn't get this question in an email since I created the PR myself